### PR TITLE
Prototype Brave SpanCollector using SpanStore

### PR DESCRIPTION
**Pull request by dsyer**
With this change the thrift server is not used by Brave (it sends spans directly top the span store). This only affects the tracing of the requests to the query service.

### CODE REVIEW

**Comment by adriancole** [comment]
so love this!

File: zipkin-java-server/src/main/resources/zipkin-server.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ server:
 spring:
   datasource:
     driver-class-name: com.mysql.jdbc.Driver
+# add this for mysql tracing: &statementInterceptors=com.github.kristofa.brave.mysql.MySQLStatementInterceptor
```
Comment (adriancole) [inline zipkin-java-server/src/main/resources/zipkin-server.yml]: do we want to suggest this? we're already tracing sql statements.

**Comment by kristofa** [comment]
Nice!

**Comment by adriancole** [comment]
I'm going to pick this in, just going to format the files and scrub the redundant tracing suggestion (since we already add annotations that the mysql tracer would)

**Comment by adriancole** [comment]
closed via ab9012af3120a340cd49f9daa998d14525be91ba thanks so much!

**Pull request closed without merging**